### DISCUSSION TITLE
clang-tidy formulae

### DISCUSSION THREAD

**Discussion by am11** [Homebrew/discussions#2015] [Writing Formulae/Casks]
Looking at the page views and votes on this issue https://stackoverflow.com/questions/53111082/how-to-install-clang-tidy-on-macos, it seems like I'm not the only one missing clang-tidy in homebrew. :)

clang-format formulae is available but clang-tidy is not. Are there any known difficulty in bottling clang-tidy up for brewing, or just a case of a missing package?

**Comment by SMillerDev** [comment]
https://github.com/Homebrew/homebrew-core/pull/61714
